### PR TITLE
Created mobile app release blog post

### PR DESCRIPTION
**Pull request by Proyoyo**
Created mobile app release blog post

### CODE REVIEW

File: _posts/2018-03-04-mobile-app-released.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+layout: post
+title:  Introducing OpenDota Mobile Application
+date: 2018-03-04
+summary: A new way to consume the data on OpenDota
+---
+The OpenDota team is proud to announce that the OpenDota mobile app is now live on the app store!
```
Comment (howardchung) [inline _posts/2018-03-04-mobile-app-released.md]: "live on the app stores for both Android and iOS!"

File: _posts/2018-03-04-mobile-app-released.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+date: 2018-03-04
+summary: A new way to consume the data on OpenDota
+---
+The OpenDota team is proud to announce that the OpenDota mobile app is now live on the app store! 
+
+Google Play Store link: https://play.google.com/store/apps/details?id=com.opendota.mobile&hl=en
+
+Apple App Store link: https://itunes.apple.com/us/app/opendota/id1354762555?ls=1&mt=8
+
+If you have any feedbacks or issues, please open an issue on [Github](https://github.com/odota/mobile/issues) or have a chat with one of us on [Discord](https://discordapp.com/invite/0o5SQGbXuWCNDcaF).
```
Comment (howardchung) [inline _posts/2018-03-04-mobile-app-released.md]: feedback

File: _posts/2018-03-04-mobile-app-released.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+Don't forget to rate the app and leave some reviews on the app store!
+
+## FAQ
+
+**Why does x feature does not exist in the mobile app?**
+
+*Our plan for the app is to reach feature parity with OpenDota website. We will enhance the experience add more features gradually.*
```
Comment (howardchung) [inline _posts/2018-03-04-mobile-app-released.md]: "and add"

**Comment by Proyoyo** [comment]
Implemented change request @howardchung

**Review by howardchung — APPROVED** [review]

**Pull request merged**